stable release version bump

joshuef opened the pull request:
…sn_networking-v0.14.2-alpha.0/sn_protocol-v0.16.2-alpha.0/sn_build_info-v0.1.7-alpha.0/sn_transfers-v0.17.2-alpha.0/sn_peers_acquisition-v0.2.9-alpha.0/sn_cli-v0.90.3-alpha.0/sn_node-v0.105.4-alpha.0/sn-node-manager-v0.7.3-alpha.0/sn_faucet-v0.4.4-alpha.0/sn_service_management-v0.2.2-alpha.0/sn_node_rpc_client-v0.6.4-alpha.0## Description

reviewpad:summary

maqi commented on sn_node/src/comm/mod.rs:
```diff
@@ -156,55 +155,62 @@ impl Comm {
+        let sessions = self.sessions.clone();
+
+        // TODO: we could cache the handles above and check them as part of loop...
+        let _handle = tokio::spawn(async move {
```
just wondering, as there is no calling of `_handle.await` to allow a waiting on this spawned task, hence the returns on the error branches, such as `return Err(Error::PeerLinkDropped);` or `Err(Error::FailedSend(peer))` actually has no effect on reporting back the errors during send?

joshuef replied:
Yup, this is the loop we need to close but without blocking in the send channel. Right now we can't send any other message until we have something to report. Instead I think we need a channel to feed back reports to the node perhaps?

maqi commented on sn_node/src/comm/peer_session.rs:
```diff
@@ -43,8 +43,7 @@ impl PeerSession {
 
-        let _ =
-            tokio::task::spawn_local(PeerSessionWorker::new(link, sender.clone()).run(receiver));
+        let _ = tokio::task::spawn(PeerSessionWorker::new(link, sender.clone()).run(receiver));
```
just wondering, shall we check all the places using `spawn_local` to confirm it's really being used properly?
I feel that for the most cases, we shall use `spawn` as default ?

joshuef commented:
Closing as #1614 and #1615 are both ahead of this

jacderida committed:
docs: clarify client::new description

Just removed a few words to try and effect another change in `sn_client`, to get a new version.

This crate has already been published and references an older version of `sn_protocol`, which is
causing a problem because `sn_cli` is referencing a newer version of it.

jacderida committed:
docs: clarify client documentation

Again, these changes are just to effect a release.

A change to the release workflow also removes testing the environment variable and re-exporting it.
On the last build I ran, it appears to have worked as expected, where previously it didn't. Doesn't
really make any sense.

joshuef committed:
chore(release): sn_auditor-v0.1.7/sn_client-v0.105.3/sn_networking-v0.14.4/sn_protocol-v0.16.3/sn_build_info-v0.1.7/sn_transfers-v0.17.2/sn_peers_acquisition-v0.2.10/sn_cli-v0.90.4/sn_faucet-v0.4.9/sn_metrics-v0.1.4/sn_node-v0.105.6/sn_service_management-v0.2.4/sn-node-manager-v0.7.4/sn_node_rpc_client-v0.6.8/token_supplies-v0.1.47
